test/integration/connection_lost_test.exs

```diff
@@ -11,15 +11,30 @@ defmodule MongoosePush.API.ConnectionTest do
     :ok
   end
 
-  # We have to skip it until MIM-748 is resolved
-  @tag :skip
   test "When connection to FCM is lost and regained a PN succeeds" do
     Tools.mock_fcm("/connection", %{"https" => false})
 
-    eventually(assert {500, ""} == Tools.post_conn_error(@url, Tools.sample_notification(:fcm)))
+    assert :not_ok == try_post(@url, Tools.sample_notification(:fcm), 5)
```

Comment:
We actually need to make sure the error is correct here (should be HTTP 503, another bug in this test).
How about:
```elixir
Enum.each(1..5, fn _ -> 
  eventually(assert {503, ""} == Tools.post_conn_error(@url, Tools.sample_notification(:fcm)))
  Process.sleep(1000)
end)
```